src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -147,11 +147,35 @@ function replaceButtonHtmlAndTooltipText(button, from, to) {
 function expandOrCollapsePanels(expandCollapseButton, panels) {
     const STRING_EXPAND = 'Expand';
     const STRING_COLLAPSE = 'Collapse';
+    const expandCollapseStudentsButton = 'a[id^="collapse-panels-button-section-"]';
+    const expandCollapseTeamButton = 'a[id^="collapse-panels-button-team-"]';
     // {@code panels} is not defined when {@code expandCollapseButton} is collapse panels button. We
     // need to define corresponding {@code targetPanels}.
-    const targetPanels = panels || $('div.panel-collapse');
 
     const isButtonInExpandMode = $(expandCollapseButton).html().trim().startsWith(STRING_EXPAND);
+
+    if (!panels && !isButtonInExpandMode) {
+        if ($(expandCollapseStudentsButton).length
+            && $(expandCollapseStudentsButton).html().trim().startsWith(STRING_EXPAND)) {
+            $(expandCollapseStudentsButton).trigger('click');
+            $(expandCollapseStudentsButton).trigger('mouseleave');
+        }
```

Comment:
Here I have tried to trigger click event and then mouse leave because after clicking button, tooltip shows up so to remove it I used "mouseleave".